src/Watchers/Warden.Watchers.ServerStatus/Extensions.cs

```diff
@@ -0,0 +1,103 @@
+﻿namespace Warden.Watchers.ServerStatus
+{
+    using System;
+    using global::Warden.Core;
+
+    public static class Extensions
+    {
+
+        /// <summary>
+        /// Extension method for adding the Port Availability watcher to the the WardenConfiguration with the default name of Web Watcher.
+        /// Uses the default HTTP GET request.
+        /// </summary>
+        /// <param name="builder">Instance of the Warden configuration builder.</param>
+        /// <param name="host">The host name for server..</param>
+        /// <param name="hooks">Optional lambda expression for configuring the watcher hooks.</param>
+        /// <returns>Instance of fluent builder for the WardenConfiguration.</returns>
+        public static WardenConfiguration.Builder AddPortAvailabilityWatcher(
+            this WardenConfiguration.Builder builder, string host,
+            Action<WatcherHooksConfiguration.Builder> hooks = null)
+        {
+            builder.AddWatcher(PortAvailabilityWatcher.Create(host), hooks);
+
+            return builder;
+        }
+
+
+        /// <summary>
+        /// Extension method for adding the Port Availability watcher to the the WardenConfiguration..
+        /// </summary>
+        /// <param name="builder">Instance of the Warden configuration builder.</param>
+        /// <param name="name">Name of the Port Availability watcher.</param>
+        /// <param name="host">A host name of server to check.</param>
+        /// <param name="hooks">Optional lambda expression for configuring the watcher hooks.</param>
+        /// <returns>Instance of fluent builder for the WardenConfiguration.</returns>
+        public static WardenConfiguration.Builder AddPortAvailabilityWatcher(
+            this WardenConfiguration.Builder builder,
+            string name, string host,
+            Action<WatcherHooksConfiguration.Builder> hooks = null)
+        {
+            builder.AddWatcher(PortAvailabilityWatcher.Create(name, host), hooks);
+
+            return builder;
+        }
+
+        /// <summary>
+        /// Extension method for adding the Port Availability watcher to the the WardenConfiguration with the default name of Web Watcher.
+        /// Uses the default HTTP GET request.
+        /// </summary>
+        /// <param name="builder">Instance of the Warden configuration builder.</param>
+        /// <param name="host">A host name of server to check.</param>
+        /// <param name="configurator">Lambda expression for configuring the WebWatcher.</param>
```

Comment:
There are some invalid names e.g. _WebWatcher_ - just look for this phrase within the whole project and replace with _PortWatcher_ :).